include/ctr/coll.h

```diff
@@ -105,9 +105,6 @@ typedef struct CollDCache
 
 #define DCACHE_COLL (*(CollDCache*) 0x1f800108)
```

Comment:
Is there no memory overlap behavior for the scratchpad within a namespace (i.e., we had unions in the OG scratchpad because several namespaces used it), but I thought even within a single namespace (e.g., coll), that there was still memory usage overlap.

Reply:
I haven't seen any overlap yet